packages/components/src/components/Markdown/Markdown.tsx

```diff
@@ -0,0 +1,79 @@
+import styled from 'styled-components';
+
+import ReactMarkdown, { Options } from 'react-markdown';
+import { variables } from '../../config';
+
+const StyledMarkdown = styled.div`
+    color: ${({ theme }) => theme.TYPE_LIGHT_GREY};
+    font-size: ${variables.FONT_SIZE.SMALL};
+    font-weight: ${variables.FONT_WEIGHT.MEDIUM};
+    line-height: 1.5;
+    padding: 0 0 32px;
+
+    h1,
+    h2,
+    h3,
+    h4,
+    h5,
+    h6 {
+        color: ${({ theme }) => theme.TYPE_DARK_GREY};
+        font-weight: ${variables.FONT_WEIGHT.DEMI_BOLD};
+    }
+
+    h1 {
+        margin: 8px 0 16px;
+        font-size: ${variables.FONT_SIZE.BIG};
+    }
+
+    h2 {
+        margin-bottom: 8px 0 12px;
+        font-size: ${variables.FONT_SIZE.NORMAL};
+    }
+
+    h3,
+    h4,
+    h5,
+    h6 {
+        margin: 4px 0 12px;
+        font-size: ${variables.FONT_SIZE.SMALL};
+    }
+
+    p,
+    ul,
+    ol {
+        margin: 4px 0 12px;
+    }
+
+    ul,
+    ol {
+        padding: 0 0 0 16px;
+    }
+
+    li {
+        margin: 0 0 8px;
+    }
+
+    a {
+        color: ${({ theme }) => theme.TYPE_GREEN};
+
+        &:hover {
+            text-decoration: underline;
+        }
+    }
+
+    img {
+        max-width: 100%;
+    }
+
+    strong {
+        font-weight: ${variables.FONT_WEIGHT.DEMI_BOLD};
+    }
+`;
+
+export const Markdown = (options: Readonly<Options>) => {
```

Comment:
Could you also add stories for this component? 🙏 
I'd like to keep stories for each DS component. Otherwise it's hard to discover

Reply:
![image](https://github.com/user-attachments/assets/7c7bf76f-f5ad-4e96-9d7c-f9cba4b02320)

Reply:
3a35749b13341f6aaf63ac3b72e973929f93bdcb